packages/ajv-validator/README.md

```diff
@@ -8,9 +10,9 @@ Define your schema with Ajv and validate incoming requests.
 No Hook:
 
 ```ts
-import { type JSONSchemaType } from 'ajv';
-import { ajvValidator } from '@hono/ajv-validator';
- 
+import { type JSONSchemaType } from 'ajv'
+import { ajvValidator } from '@hono/ajv-validator'
+
```

Comment:
Looks like prettier and vscode decided to format code blocks on save.. happy to revert this if it's an issue

Reply:
It's okay!